doc/DPOWchain.md

```diff
@@ -0,0 +1,88 @@
+### Howto setup DPOW chain and notarizer exec
+
+```bash
+git clone https://github.com/jl777/komodo.git dexkomodo
+cd ~/dexkomodo
+git checkout DEX
+./zcutil/build.sh -j8
```

Comment:
wouldn't "hardset" the number of compilation processes/jobs - merge/approval waits for https://github.com/CHMEX/dPoW/pull/1